.github/workflows/release.yml

```diff
@@ -14,11 +14,10 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: "18.13.0"
-      - name: Install dependencies
+      - name: Install dependencies and publish
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
         run: |
           npm ci
           npm run build
-          npm --no-git-tag-version version from-git
           npm publish --access=public
```

Comment:
should this not now get the version from --tag? how do we control which version is published & what triggers the action?

Reply:
The action would be triggered by creating a new Release, the version should come from the version in package.json like the [react-hooks](https://github.com/ably-labs/react-hooks/blob/main/.github/workflows/npm-publish.yml) package, I believe.

Reply:
but react-hooks has no releases or tags? https://github.com/ably-labs/react-hooks/releases

Reply:
Sorry, I meant; the version comes from the package.json in `react-hooks`. But I have added the --tags argument in the publish step.